corehq/apps/translations/app_translations.py

```diff
@@ -6,11 +6,13 @@
 import itertools
 import re
 import ghdiff
+import openpyxl
```

Comment:
nit: imports should generally follow https://www.python.org/dev/peps/pep-0008/#imports meaning ghdiff and openpyxl would go where six is and io would rise up here. (would be nice to get isort functionality for stickler)

Reply:
thanks for pointing that. Will update.